CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Bumped clustertest to `v0.1.1` to fix `GITHUB_TOKEN` issue
 
+### Added
```

Comment:
Not sure why this is showing up as a diff, maybe rebase the branch on master?

Reply:
The rebase isn't going to be clean and will need a lot of work to finish off this whole PR. I just wanted to get feedback on the approach before I did all that work in case it wasn't desired :)

Reply:
I've rebased. The diff is much more readable now.